cl/opinion_page/views.py

```diff
@@ -354,28 +354,9 @@ def view_opinion(request, pk, _):
         favorite_form = FavoriteForm(instance=fave)
 
     if not is_bot(request):
-        # Get the citing results from Solr for speed. Only do this for humans
-        # to save on disk usage.
-        conn = sunburnt.SolrInterface(settings.SOLR_OPINION_URL, mode="r")
-        q = {
-            "q": "cites:({ids})".format(
-                ids=" OR ".join(
-                    [
-                        str(pk)
-                        for pk in (
-                            cluster.sub_opinions.values_list("pk", flat=True)
-                        )
-                    ]
-                )
-            ),
-            "rows": 5,
-            "start": 0,
-            "sort": "citeCount desc",
-            "caller": "view_opinion",
-        }
-        citing_clusters = conn.raw_query(**q).execute()
+        children = cluster.children_with_data
     else:
-        citing_clusters = None
+        children = None
```

Comment:
This sounds dumb, but can we improve this for bots? What would happen if we cached the values the humans got the first time and then later showed those values to the bots? So bots can't generate data, but if humans already have, then the bots can access it? If we cache to the DB, it'll be fast and take up negligible space? Perhaps a tweak for later. It'd be good for SEO (something I don't want to care about, but sadly do).

Reply:
Relevant changes reverted in f85e738, but perhaps this comment should be preserved for further discussion

Reply:
Moved to: https://github.com/freelawproject/courtlistener/issues/1197